### ISSUE TITLE
Failed converting result to metric: max_standby_streaming_delay

### ISSUE THREAD

**Issue by saifulmuhajir** [wrouesnel/postgres_exporter#60]
This is actually weird error and only happened on our **master server with three slaves.** The process runs fine but there's this error that keeps going:

```
ERRO[1747] Failed converting result to metric: max_standby_streaming_delay [49 109 105 110] time: unknown unit min in duration 1min  source=postgres_exporter.go:590
```

Is this because of the number of slaves (three)? Is there anything I can modify to make this error go away.

**Comment by wrouesnel** [comment]
It's "min" not parsing as a duration it looks like. Which makes sense since golang parsing doesn't accept that, but I guess it can be generated by postgres?

I need to to implement some looser time parsing to conform to what postgres actually returns.

**Comment by saifulmuhajir** [comment]
Strangely though, this happened only on one particular server. 

If you need more information, I'll be glad to provide the best I can.

**Comment by saifulmuhajir** [comment]
It turns out that the parameter **max_standby_streaming_delay** which set to 1min, **60000** cannot be converted. Postgres sent the parameter value in **1min** and the exporter cannot handle the value. 

I guess you should add `min` or `s` as the metric unit.

**Comment by wrouesnel** [comment]
Good to know. Time to go spelunk the postgres time documentation.

**Comment by mattbostock** [comment]
I think it'd make sense to use the [pg_settings view](https://www.postgresql.org/docs/current/static/view-pg-settings.html) for this - it doesn't require `SHOW`, returns integers with no suffixes to parse, and includes the data type and a description.

Example:
```
# SELECT * FROM pg_settings WHERE name = 'max_standby_streaming_delay';
-[ RECORD 1 ]----------------------------------------------------------------------------------------------------------
name       | max_standby_streaming_delay
setting    | 300000
unit       | ms
category   | Replication / Standby Servers
short_desc | Sets the maximum delay before canceling queries when a hot standby server is processing streamed WAL data.
extra_desc |
context    | sighup
vartype    | integer
source     | configuration file
min_val    | -1
max_val    | 2147483647
enumvals   |
boot_val   | 30000
reset_val  | 300000
sourcefile | /etc/postgresql/9.4/main/postgresql.conf
sourceline | 252
```

The `short_desc` could be returned as the 'help' description for the metric.

**Comment by wrouesnel** [comment]
Fixed by 98ba5663221a514b0d334780eb3651b9838c42c7

(which merges your PR)
